spec/GraphQLOverHTTP.md

```diff
@@ -237,120 +311,304 @@ omitted.
 
```

Comment:
(Previous line) Why not mention how `operationName` is encoded outside of the Note?